### DISCUSSION TITLE
Network commands via SSH won't connect.

### DISCUSSION THREAD

**Discussion by teemue** [RaspAP/raspap-webgui#966] [General]
RasPi won't success in any out-of-lan access attempts. This worked before and the configuration has stayed same.
RP is connected to the router via cable and access point is setup via wlan.

```
pi@raspiap:~ $ sudo apt update
Err:1 http://archive.raspberrypi.org/debian buster InRelease
   Temporary failure resolving "archive.raspberrypi.org"
Err:2 http://raspbian.raspberrypi.org/raspbian buster InRelease
   Temporary failure resolving "raspbian.raspberrypi.org"
pi@raspiap:~ $ ping google.com
ping: google.com: Temporary failure in name resolution
```
VPN is not on.
If I remove/empty /etc/resolv.conf, it works again until restart.

**Comment by Bostonvex** [comment]
Error message indicates that it is a DNS issue and as stated by you if you remove/empty /etc/resolv.conf it works.

**By default RaspAP also uses its upstream DHCP server's name servers.**

1. What DNS server did you set in your DHCP config?
2. What does resolv.conf look like when its broken?
2. What does resolv.conf look like when it works?
3. Do you have a local DNS server?
4. Which externals do you forward to?
5. Provide output of netstat -nr
6. When name resolution breaks, please ping 8.8.8.8 to confirm you have internet access
7. Install dnsutils if you don't have it already `pi@raspberrypi:~ $ sudo apt-get install bind-utils`
8. While experiencing the issue, use nslookup or dig to manually set your DNS server and test if resolution works like below:

```
pi@raspberrypi:~ $ nslookup
> server 10.112.0.1
Default server: 10.112.0.1
Address: 10.112.0.1#53
> www.cnn.com
Server:         10.112.0.1
Address:        10.112.0.1#53

Non-authoritative answer:
www.cnn.com     canonical name = turner-tls.map.fastly.net.
Name:   turner-tls.map.fastly.net
Address: 151.101.1.67
Name:   turner-tls.map.fastly.net
Address: 151.101.65.67
Name:   turner-tls.map.fastly.net
Address: 151.101.129.67
Name:   turner-tls.map.fastly.net
Address: 151.101.193.67
Name:   turner-tls.map.fastly.net
Address: 2a04:4e42::323
Name:   turner-tls.map.fastly.net
Address: 2a04:4e42:200::323
Name:   turner-tls.map.fastly.net
Address: 2a04:4e42:400::323
Name:   turner-tls.map.fastly.net
Address: 2a04:4e42:600::323
> exit

pi@raspberrypi:~ $
```

**Reply by teemue** [reply]
I reinstalled (well, over-installed) RaspAP and everything seems to work well now. If I face the same problem again, I'll follow these steps and report.
